features/0496-transition-to-oob/README.md

```diff
@@ -0,0 +1,251 @@
+# Aries RFC 0496: Transition to the Out of Band Protocol
+
+- Authors: [Stephen Curran](mailto:swcurran@cloudcompass.ca)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-06-10
+- Status Note: In step 0 - community is reviewing the proposed approach. **Target Completion Date: 2020.06.15**
+- Supersedes:
+- Start Date: 2020-06-07
+- Tags: feature, community-update
+
+## Summary
+
+The Aries community has agreed to transition from using the `invitation` messages in [RFC 0160 Connections](../0160-connection-protocol/README.md)
+and [RFC 0023 DID Exchange](../0023-did-exchange/README.md) to using the plaintext `invitation` message in [RFC 0434 Out of Band](../0434-outofband/README.md).
```

Comment:
Although aries-framework-go added support for did-exchange long ago, I'm not sure the rest of the community has, and thus I cast doubt on this claim that includes RFC0023. This RFC is not part of RFC0302.

What is the plan to transition the community from RFC0160 to RFC0023 ?

Reply:
Renamed the RFC and added language to clarify that this also moves the community to DID Exchange.

Also moved the date out 6 months, since we are already 2.5 months over the plan to have this PR merged, let alone acted on.